config/authors/murhafsousli.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "MurhafSousli",
+  "name": "Murhaf Sousli",
   "bio": "",
```

Comment:
Fullstack Software Engineer - Angular enthusiast. Interested in AI and Machine learning.